### ISSUE TITLE
v0.9.2 release preparation

### ISSUE THREAD

**Issue by namhyung** [namhyung/uftrace#643]
v0.9.2
=====
I'm thinking about releasing a new version (v0.9.2) soon.  Although there are still some bugs here and there I think it's time to release before moving to work on other features.  It'd be nice if you could help testing current version.

This is a minor release with following changes

tracing library calls without PLT (#592)
------------------------------------------
modern distro started to build system binaries without PLT (gcc has `-fno-plt`) it made uftrace failed to trace library calls.  this version added PLT equivalent around the text segment to trace them.

add signal trigger (#340)
---------------------------------
sometimes it's hard to determine which function we want to control.  also daemon programs don't exit so uftrace cannot know when it should stop tracing.  the signal triggers add a way to control uftrace behavior using signals.

parallel execution of tests (#569)
----------------------------------
test program (`tests/runtest.py`) was enhaned to use multi-process facility of python so that it can run tests in parallel.  it improved speed of running tests during development.

rudimentary support for Rust programs (#594)
------------------------------------
recently Rust compiler added an instrumentation option to add `mcount` call for each function.  uftrace can trace those binaries like C/C++ if it's compiled with `-Z instrument-mcount`.

show data symbols in arguments (#632)
--------------------------------
this work is not finished yet but it enabled data symbol loading which is used to display arguments and return values and it'll help understanding user's program.

And many bugfixes..

**Comment by honggyukim** [comment]
Thanks for the summary. It'd be better to make a note if listed features are merged into `master. For example, signal feature is not yet merged so people may get confused.

**Comment by namhyung** [comment]
Done.

**Comment by namhyung** [comment]
Merged: 66fc1fb9
